packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/FormDescriptionEditorEventFragment.types.ts

```diff
@@ -95,7 +95,7 @@ export interface GQLAddWidgetPayload {
   __typename: string;
 }
 
-export interface GQLAddWidgetSuccessPayload extends GQLAddWidgetPayload {
+export interface GQLSuccessPayload extends GQLAddWidgetPayload {
```

Comment:
Shouldn't we go further in the simplification here, and have a single type?
Something like:

```
export interface GQLWidgetOperationPayload {
  __typename: string;
}

export interface GQLSuccessPayload extends GQLWidgetOperationPayload {
  id: string;
}

export interface GQLErrorPayload extends GQLWidgetOperationPayload {
  message: string;
}
```

It feels strange/wrong to have 9 definitions of `GQLSuccessPayload` in the same file.